ui/app/components/dashboard/quick-actions-card.js

```diff
@@ -0,0 +1,182 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
+
+/**
+ * @module DashboardQuickActionsCard
+ * DashboardQuickActionsCard component allows users to see a list of secrets engines filtered by
+ * kv, pki and database and perform certain actions based on the type of secret engine selected
+ *
+ * @example
+ * ```js
+ * <DashboardQuickActionsCard />
+ * ```
+ * @param {array}
+ */
+
+const getActionsByEngineType = (type) => {
+  switch (type) {
+    case 'kv':
+      return [
+        {
+          actionTitle: 'Find KV secrets',
```

Comment:
So this is a design question, but when no secret engine is selected, do they want the action to be empty / disabled / or is the intention to show "Find KV secrets". 
![image](https://github.com/hashicorp/vault/assets/6618863/f442ca04-0bd2-4596-951e-aab98416780e)

Reply:
From the current designs, it is intentional to show "Find KV secrets" as the default quick actions state.
<img width="455" alt="Screenshot 2023-08-01 at 11 52 01 AM" src="https://github.com/hashicorp/vault/assets/30884335/006d2aaf-69a3-4742-911e-95cd18ad9af7">

I will confirm with design if we want to disable it!